src/router/index.js

```diff
@@ -54,7 +54,7 @@ function createRouter() {
           { path: 'userpassword', component: ProfilePasswordView }
         ]
       },
-      { path: '/blog/index', component: BlogIndex },
+      { path: '/blogs', component: BlogIndex },
```

Comment:
modified frontend path here as well for the frontend

Reply:
I've been debating our use of plural "blogs" here, but then I considered that "blog" is the model name, and the index being "blogs" would be like Rails convention. Even though we're not using rails here, I do think naming the index page "blogs" makes sense.